### DISCUSSION TITLE
How long should pushkin prep run?

### DISCUSSION THREAD

**Discussion by benthebear** [l3atbc-datadog/pushkin#194] [General] [answered]
Heya there … 

… I wanted to test Pushkin in my local Mac. Everything from the Tutorial went fine, but `pushkin prep` just won't finish on my command line. I kept it running for two hours and it didn't add new files to the front-end Folder after 10 or 20 minutes. It just didn't stop echoing `...` 

Is that common that it takes so long? Shall I wait any longer?

Thanks!

**Comment by hzeng2** [comment]
Hi there! Thanks for letting us know about this issue you are having. Currently we are updating and testing the environment for Pushkin. We will update here when we work out a solution for you. And thanks for typing up details about the issue!

**Reply by benthebear** [reply]
Thanks for the quick reply and giving me an update on the reasons!

**Comment by jkhartshorne** [comment, marked as answer]
@benthebear -- this seems to be a bug that cropped up in a recent version of the CLI. This is why we need to write tests. I started an issue: https://github.com/pushkin-consortium/pushkin/issues/195

**Reply by benthebear** [reply]
Thanks! I'm always happy to hear, that I'm not to stupid to follow the tutorial. :D

**Reply by jkhartshorne** [reply]
Please check the aforementioned issue. If you can check to see if the solution works for you, that would be super helpful.

Note that I've now got an issue with the next step (`pushkin start`), which I suspect is due to updates to Docker. But one thing at a time!

**Reply by jkhartshorne** [reply]
@benthebear you are tagged on the issue I created from this discussion (https://github.com/pushkin-consortium/pushkin/issues/195) but I think the direct mentions of you there may not actually ping you. 

Anyway, if you look over at the Issues, you'll see a bunch of activity. Trying to address this issue meant upgrading packages with breaking changes that depend on other packages with other breaking changes and ... well, it's been a whole thing. I think we are getting there, though.
